index.js

```diff
@@ -1,62 +1,113 @@
 'use strict';
 
 var postcss = require('postcss');
+var R = require('ramda');
 
 module.exports = postcss.plugin('postcss-responsive-type', function () {
 
   // Default parameters
-  var params = {
-    minSize: '12px',
-    maxSize: '21px',
-    minWidth: '420px',
-    maxWidth: '1280px'
+  var defaultParams = {
+    'font-size': {
+      minSize: '12px',
+      maxSize: '21px',
+      minWidth: '420px',
+      maxWidth: '1280px'
+    },
+    'line-height': {
+      minSize: '1rem',
+      maxSize: '2rem',
+      minWidth: '420px',
+      maxWidth: '1280px'
+    },
+    'letter-spacing': {
+      minSize: '1rem',
+      maxSize: '2rem',
+      minWidth: '420px',
+      maxWidth: '1280px'
+    }
+  };
+
+  // Map supported params to their range declarations
+  var paramRangeDecl = {
+    'font-size': 'font-range',
+    'line-height': 'line-height-range',
+    'letter-spacing': 'letter-spacing-range'
   };
 
   // Map expanded declarations to params
   var paramDecls = {
-    minSize: 'min-font-size',
-    maxSize: 'max-font-size',
-    minWidth: 'lower-font-range',
-    maxWidth: 'upper-font-range'
+    'font-size': {
+      minSize: 'min-font-size',
+      maxSize: 'max-font-size',
+      minWidth: 'lower-font-range',
+      maxWidth: 'upper-font-range'
+    },
+    'line-height': {
+      minSize: 'min-line-height',
+      maxSize: 'max-line-height',
+      minWidth: 'lower-line-height-range',
+      maxWidth: 'upper-line-height-range'
+    },
+    'letter-spacing': {
+      minSize: 'min-letter-spacing',
+      maxSize: 'max-letter-spacing',
+      minWidth: 'lower-letter-spacing-range',
+      maxWidth: 'upper-letter-spacing-range'
+    }
   };
 
   var rootSize = '16px';
 
-  /**
-   * Fetch plugin parameters from css rules
-   * @param  {object} rule CSS rule to parse
-   */
-  var fetchParams = function(rule){
-
-    // Fetch params from shorthand font-size
-    rule.walkDecls('font-size', function(decl){
+  var fetchResponsiveSizes = function(rule, declName, cb){
+    rule.walkDecls(declName, function(decl){
 
       if (decl.value.indexOf('responsive') > -1) {
-        var vals = decl.value.match(/\d*\.?\d+\w+/g);
-        if (vals){
-          params.minSize = vals[0];
-          params.maxSize = vals[1];
+        var vals = decl.value.match(/\d*\.?\d+(?:\w+)?/g);
+        if (vals) {
+          cb(vals[0], vals[1]);
         }
       }
 
     });
+  };
 
-    // Fetch params from shorthand font-range
-    rule.walkDecls('font-range', function(decl){
+  var fetchRangeSizes = function(rule, declName, cb){
+    rule.walkDecls(declName, function(decl){
       var vals = decl.value.split(/\s+/);
-      params.minWidth = vals[0];
-      params.maxWidth = vals[1];
+      cb(vals[0], vals[1]);
       decl.remove();
     });
+  };
+
+  /**
+   * Fetch plugin parameters from css rules
+   * @param  {object} rule CSS rule to parse
+   */
+  var fetchParams = function(rule, declName){
+    var params = R.clone(defaultParams[declName]);
```

Comment:
Is this the only place Ramda is used? Pretty bulky dep just for a clone func, especially when we don't actually need a deep clone here.